minique/api.py

```diff
@@ -1,5 +1,8 @@
 import json
 import time
+from typing import Callable, Optional, Union
+
+from redis import Redis
```

Comment:
Why the new lines between imports..?

Reply:
It's [`isort`](https://github.com/timothycrosley/isort) custom to split groups of imports:

* standard library
* third-party (installed packages)
* first-party (current package / project)